R/coveralls.R

```diff
@@ -57,9 +71,45 @@ to_coveralls <- function(x, service_job_id = Sys.getenv("TRAVIS_JOB_ID"),
       "source_files" = res)
   }
 
+  if (!is.null(git_info)) payload <- c(payload, git = list(git_info))
```

Comment:
Github hid some diff lines. But payload will always exist with some defaults, and your coverage will be reported to coveralls (just without git info)
```r
payload <- if (is.null(repo_token)) {
    list(
      "service_job_id" = jsonlite::unbox(service_job_id),
      "service_name" = jsonlite::unbox(service_name),
      "source_files" = res)
  } else {
    list(
      "repo_token" = jsonlite::unbox(repo_token),
      "source_files" = res)
  }
```